docs/basics.rst

```diff
@@ -136,7 +136,13 @@ On Linux, the above example can be compiled using the following command:
 
 .. code-block:: bash
 
-    $ c++ -O3 -Wall -shared -std=c++11 -fPIC `python3 -m pybind11 --includes` example.cpp -o example`python3-config --extension-suffix`
+    $ c++ -O3 -Wall -shared -std=c++11 -fPIC $(python3 -m pybind11 --includes) example.cpp -o example$(python3-config --extension-suffix)
+
+.. note::
+
+    If you used :ref:`include_as_a_submodule` to install the pybind11 source,
```

Comment:
"install" is a bit weird here, perhaps (since you're just checking out the source, not installing it in a specific location), but it's under the section "Installing the library", so... yeah, fine with me.

Reply:
"Get" is probably better than "install" here.